trieste/models/gpflow/optimizer.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 The Trieste Contributors
```

Comment:
can you remove this file please, this should be part of the optimizer refactoring PR